com.unity.ml-agents/Tests/Editor/ModelRunnerTest.cs

```diff
@@ -0,0 +1,100 @@
+using NUnit.Framework;
+using UnityEngine;
+using UnityEditor;
+using Barracuda;
+using MLAgents.InferenceBrain;
+using MLAgents.Sensors;
+using System.Linq;
+
+namespace MLAgents.Tests
+{
+    [TestFixture]
+    public class ModelRunnerTest
+    {
+        const string k_continuous2vis8vec2actionPath = "Packages/com.unity.ml-agents/Tests/Editor/Resources/continuous2vis8vec2action.nn";
```

Comment:
Personal taste, but I'd lean towards shorter names and comments, e.g.
```
/// Continuous model: 2 visual observations, 8 vector observations, 2 continuous actions
const string k_continuousModelPath = "Packages/com.unity.ml-agents/Tests/Editor/Resources/continuous2vis8vec2action.nn";
```
It's just tests, though, so don't worry much.

Reply:
👍